parser/parser_test.go

```diff
@@ -1223,3 +1223,21 @@ func TestParse(t *testing.T) {
 		})
 	}
 }
+
+func TestExpressionSizeCodePointLimit(t *testing.T) {
+	p, err := NewParser(Macros(AllMacros...), ExpressionSizeCodePointLimit(-2))
+	if err == nil {
+		t.Fatal("expected NewParser to return an error when given an invalid expression size code point limit")
+	}
+	p, err = NewParser(Macros(AllMacros...), ExpressionSizeCodePointLimit((2)))
+	if err != nil {
+		t.Fatal(err)
+	}
+	_, errs := p.Parse(common.NewTextSource("foo"))
+	if got, want := len(errs.GetErrors()), 1; got != want {
+		t.Fatalf("expected parsing to result in exactly %d errors, got %d errors", want, got)
```

Comment:
Usually, you want to phrase the error messages as `got X, wanted Y` rather than the other way around. Here and below.

Reply:
Done.